Makefile

```diff
@@ -93,6 +93,11 @@ go.test.unit: $(KUBEBUILDER)
 manifests: vendor kubebuilder.manifests
 	@$(INFO) Generating CRD manifests
 	$(CONTROLLERGEN) crd:maxDescLen=0,trivialVersions=true paths=./apis/stacks/... output:dir=$(CRD_DIR)
+# Add "helm.sh/hook: crd-install" and "helm.sh/hook-delete-policy: before-hook-creation" annotations for clusterstackinstalls and stackinstalls CRDs
+	$(eval TMPDIR := $(shell mktemp -d))
+	kustomize build cluster/charts -o $(TMPDIR)
```

Comment:
all tools in our build flow should be known versions, this appears to use whatever `kustomize` happens to be on the developers machine.  also, it may fail completely if the dev doesn't already have kustomize installed.

For example, you can see how the build submodule installs a specific version of `kubebuilder` into the common tools cache, and will reference that specific version throughout the build, so we are always using a known good version across everyone's machines.  The goal here is consistent and repeatable builds.
https://github.com/upbound/build/blob/master/makelib/kubebuilder.mk#L83

Reply:
Yes, this makes sense, thanks for the information.

Here is the PR adding kustomize: https://github.com/upbound/build/pull/76